bin/lib/start.js

```diff
@@ -24,8 +24,9 @@ module.exports = function (program, server) {
 
   start.action((opts) => {
     let argv = extend({}, opts, { version: program.version() })
+    let configFile = argv['configFile'] || './config.json'
```

Comment:
Might want to use path.join here with __dirname is that seems to be consistent with much of the code base

e.g. https://github.com/solid/node-solid-server/blob/dea121afbb92dd8e9ef8929d1d09246cd5ba5180/test/integration/capability-discovery-test.js#L12

Reply:
I'm happy to do that; was just making a minimal change to master. Does that buy us anything over just a relative path to config.json? I made pretty much the reverse change in shex.js when I made it able to load examples from a URL I suspect the only change is for node-solid-server is in debugging output.

Reply:
No, looks good, happy to merge!